src/RoomsSDKAdapter.test.js

```diff
@@ -0,0 +1,59 @@
+import {isObservable} from 'rxjs';
+
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter Interface', () => {
+  let roomsSDKAdapter;
+  let mockSDK;
+
+  beforeEach(() => {
+    mockSDK = {
+      rooms: {
+        get: jest.fn(),
+      },
+    };
+
+    roomsSDKAdapter = new RoomsSDKAdapter(mockSDK);
+  });
+
+  describe('#getRoom()', () => {
+    test('returns an observable', () => {
+      expect(isObservable(roomsSDKAdapter.getRoom())).toBeTruthy();
+    });
+
+    test('returns a room in a proper shape', (done) => {
+      const mockSDKRoom = {
+        id: 'abc',
+        type: 'group',
+        title: 'mock room',
+      };
+
+      mockSDK.rooms.get = jest.fn(() => Promise.resolve(mockSDKRoom));
+
+      roomsSDKAdapter.getRoom('id').subscribe((room) => {
+        expect(room.ID).toBe(mockSDKRoom.id);
```

Comment:
```js
expect(room).toEqual(mockSDKRoom)
```

perhaps?

Reply:
The shape of the data that is returned from the SDK has `id` while the adapter expects `ID`. You can see in the implementation that we have to shape the data to match the adapter requirements so we can't do a direct comparison between the sdk object and the adapter object.

Reply:
I was just a bit hesitant about having multiple assertions in one test. However, I did some researches and learned that as long we follow this `Arrange, Act, Assert` pattern, it is fine to have multiple assertions. more infor: http://wiki.c2.com/?ArrangeActAssert